src/MessagingBubble/NormalMessagingBubble.less

```diff
@@ -86,39 +86,79 @@
   .margin--left--xs();
 }
 
-Button.NormalMessagingBubble--Message--ActionButton--Own {
-  width: 100%;
-  text-align: right;
+.NormalMessagingBubble--Message--Delete {
+  display: none;
+  color: @ERROR_RED;
+  .text--semi-bold();
+  line-height: @size_l;
+}
 
-  &:focus,
-  &:active {
+// This class may be identical in content to the --Timestamp class,
+// but this one is not rendered invisible by bubble-hover behavior.
+.NormalMessagingBubble--Message--Timestamp--NonInteractive {
+  color: @neutral_medium_gray;
+  font-size: 0.875rem;
+  font-weight: normal;
+  .text--line-height-1();
+  white-space: nowrap;
+  flex-shrink: 0;
+  min-width: 3.5rem;
+}
+
+.NormalMessagingBubble--Message--Timestamp {
```

Comment:
can this be merged with `NormalMessagingBubble--Message--Timestamp--NonInteractive`?

like `NormalMessagingBubble--Message--Timestamp--NonInteractive, .NormalMessagingBubble--Message--Timestamp`